<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Details</class>
 <widget class="QWidget" name="Details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: #242A38;
	border: 2px solid white;
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="field_path">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_creation">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 25px;
}</string>
     </property>
     <property name="text">
      <string>Created:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="field_creation">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_owner">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 25px;
}</string>
     </property>
     <property name="text">
      <string>Owner:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_details">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 25px;
qproperty-alignment: 'AlignHCenter|AlignVCenter';
}</string>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="field_owner">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_fileName">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 25px;
}</string>
     </property>
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_path">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 25px;
}</string>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="field_fileName">
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid none;
color: white;
font-size: 20px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btn_close">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background: #f76163;
    border-width: 1px;
    border-radius: 13px;
    border-color: #f76163;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 white);
}

QPushButton:pressed {
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 black);
}

QPushButton:focus {
   border: none;
   outline: none;
}</string>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
